packages/manager/modules/pci/src/components/project/users/users.html

```diff
@@ -215,8 +215,16 @@
         ></span>
     </oui-button>
 
+    <div data-ng-if-start="$ctrl.showRolesMatrix" class="mt-2">
+        <oui-message data-type="info" class="d-inline-block">
+            <span
+                data-translate="pci_projects_project_users_high_perf_info"
+            ></span>
+        </oui-message>
+    </div>
+
     <pci-projects-project-users-matrix
         data-roles="$ctrl.roles"
-        data-ng-if="$ctrl.showRolesMatrix"
+        data-ng-if-end
```

Comment:
`ngIfStart/End` are not really used in our codebase. 
IMHO it makes code difficult to read.

Reply:
**question(non-blocking):** @y4nnL Can we take this into in account?